docs/sources/installation/debian.md

```diff
@@ -43,6 +43,8 @@ Add our GPG key to install signed packages:
 wget -q -O - https://packages.grafana.com/gpg.key | sudo apt-key add -
 ```
 
+#### Setup repo using add-apt-repository
```

Comment:
The user has to do either the "Setup repo using add-apt-repository" or the "Set up repo for ARM" part. The rest of the setup is the same (both before and after). 

Is it better to duplicate all of the instructions that are the same instead of using sub-headers? I wish we had a way to showing one one option at a time and having it be possible to switch between them.

Reply:
Ugh. This is another major argument in favor of the installation script: https://github.com/grafana/grafana/issues/20939

We want to keep the flow of the instructions as clear as possible and try to avoid confusing users with something they might not need to do. For example, we want to make clear that they do either one setup or the other, not both. Also, we do not want them to ignore the small section that follows it.

Reply:
Before I make the final changes. Should I duplicate everything between `## 1. Download and install` and `### Set up repository for ARM` for the ARM part?

Reply:
Not right now. I have another change coming to this topic in the near future, I will take another look then.

Also, be sure to change "Setup" to "Set up"

Reply:
I thought I had done that already, sorry about that.

Reply:
It is actually changed already.